src/tags/object/Image.js

```diff
@@ -297,10 +299,23 @@ const Model = types.model({
 
   // @todo the name is for backward compatibility; change the name later
   get _value() {
-    const value = self.parsedValue;
+    let value = self.parsedValue;
 
-    if (Array.isArray(value)) return value[self.currentImage];
-    return value;
+    if (Array.isArray(value)) value = value[self.currentImage];
+
+    if (!self.imageCrossOrigin) {
+      return value;
+    } else if (isFF(FF_DEV_4081) && /^https?:.*?s3\.amazonaws\.com\//g.test(value)){
```

Comment:
Is this strictly an issue with AWS S3? And are there any other cases beyond that can trigger this problem?

Reply:
It's a known issue with just AWS S3 not implementing the CORS spec correctly.